docker/dockerfiles/toolchains/toolchains-centos7.Dockerfile

```diff
@@ -2,8 +2,8 @@
 # NOTE: build context MUST be set to `docker/dockerfiles/toolchains/`
 #  DOCKER_BUILDKIT=1 docker build --rm=true -f docker/dockerfiles/toolchains/toolchains-centos7.Dockerfile -t toolchains-centos7:latest docker/dockerfiles/toolchains/
 
-ARG GCC_INSTALL_HOME=/opt/rh/gcc-toolset-10/root/usr
-ARG GCC_DOWNLOAD_URL=https://ftp.gnu.org/gnu/gcc/gcc-10.3.0/gcc-10.3.0.tar.gz
+ARG GCC_INSTALL_HOME=/opt/rh/gcc-toolset-14/root/usr
```

Comment:
this may not work well with clang-14, clang-14 only recognises gcc-toolset up to 11